### PR TITLE
Add Tigris Vector Store

### PR DESCRIPTION
**Pull request by ovaistariq**
**Tigris VectorStore**
Integration with the Tigris VectorStore.

More details about the Tigris VectorStore can be found [here](https://www.tigrisdata.com/docs/concepts/vector-search/).

**Note:**
This is my first contribution to this repo. I have tried to follow the contribution guide. If I missed something, please let me know.

### CODE REVIEW

File: docs/docs/modules/indexes/vector_stores/integrations/tigris.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
```
Comment (jacoblee93) [inline docs/docs/modules/indexes/vector_stores/integrations/tigris.md:1]: Can you change this to a `.mdx` file and move the code blocks to `/examples`?

You can read more about it here:

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#documentation-and-integration-tests

**Comment by jacoblee93** [comment]
Looks great overall! See feedback.

**Comment by ovaistariq** [comment]
> Looks great overall, apologies for the delay! See feedback.

No worries, I can understand. I have addressed all the feedback, could you take a look again?

**Comment by jacoblee93** [comment]
Merged the extension PR #1423, closing this. Thank you!

**Pull request closed without merging**